app/enterprise/1.3-x/plugin-development/custom-logic.md

```diff
@@ -263,34 +263,54 @@ Plugin                      | Priority
 ----------------------------|----------
 pre-function                | `+inf`
 zipkin                      | 100000
-ip-restriction              | 3000
+exit-transformer            | 9999
 bot-detection               | 2500
 cors                        | 2000
 session                     | 1900
+oauth2-introspection        | 1700
+mtls-auth                   | 1006
 kubernetes-sidecar-injector | 1006
 jwt                         | 1005
+degraphql                   | 1005
 oauth2                      | 1004
 key-auth                    | 1003
+key-auth-enc                | 1003
 ldap-auth                   | 1002
+ldap-auth-advanced          | 1002
 basic-auth                  | 1001
+openid-connect              | 1000
 hmac-auth                   | 1000
+request-validator           | 999
```

Comment:
request-validator actually has two values for the priority config.  one is 999, another is 200 in the code of the plugin.
I believe 999 should be correct.